src/Arcus.WebApi.Logging/RequestTrackingMiddleware.cs

```diff
@@ -112,10 +112,12 @@ private static async Task<string> SanitizeRequestBodyAsync(Stream requestStream)
                 return null;
             }
 
+            long originalPosition = 0;
             if (requestStream.CanSeek)
             {
                 if (requestStream.Position != 0)
```

Comment:
I don't know if I'd check if the current position is not 0, before setting the `originalPosition` and positioning the stream to the beginning.   But that's just my opinion

Reply:
Good point!

I'll assign the original position regardless of the position